internal/isoeditor/rhcos.go

```diff
@@ -0,0 +1,273 @@
+package isoeditor
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"regexp"
+
+	"github.com/carbonin/assisted-image-service/internal/isoutil"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	RamDiskPaddingLength  = uint64(1024 * 1024) // 1MB
+	IgnitionPaddingLength = uint64(256 * 1024)  // 256KB
+	ignitionImagePath     = "/images/ignition.img"
+	ramDiskImagePath      = "/images/assisted_installer_custom.img"
```

Comment:
This is what we will be asking the assisted-service for right? Wonder if it should just be 
```suggestion
	ramDiskImagePath      = "/images/ramdisk.img"
```

Reply:
:man_shrugging: This isn't a new name and I'm not sure it really matters